stock_dock/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    "name": "Loading Dock",
+    "summary": "Manage the loading docks of your warehouse.",
+    "version": "18.0.1.0.1",
```

Comment:
Normally, you should reset this

```suggestion
    "version": "18.0.1.0.0,
```

Reply:
Thanks for your time.